tests/text_preparation/test_normalizers.py

```diff
@@ -135,3 +143,352 @@ def test_nfkc_normalization(nfkc_normalizer, sentence, expected_normalized_text)
     """ Testing NFKC normalization."""
     normalized_text = nfkc_normalizer.normalize(sentence)
     assert normalized_text == expected_normalized_text
+
+
+def test_lowercase_normalization():
+    assert Lowercase().normalize("Hello, How Are You?") == "hello, how are you?"
+
+
+def test_get_regex_normalizers():
+
+    regex_norm_rule = {"pattern": ".*", "replacement": ""}
+    regex_normalizer = RegexNormalizerRuleFactory.get_regex_normalizers([regex_norm_rule])[0]
+    assert regex_normalizer.normalize("Cisco") == ""
+
+
+# TESTING NORMALIZER CLASSES
+
+
+def _check_match(text_preparation_pipeline, regex_norm_rule, input_text, expected_text):
+
+    regex_normalizer = RegexNormalizerRuleFactory.get_default_regex_normalizer_rule(
+        regex_norm_rule
+    )
+    text_preparation_pipeline.normalizers = [regex_normalizer]
+    normalized_text = text_preparation_pipeline.normalize(input_text)
+    assert normalized_text == expected_text
+
+
+@pytest.mark.parametrize(
```

Comment:
Could you also do some tests against japanese for the preprocessors and normalizers?